AlembicImporter/Assets/UnitTests/AlembicExportTest.cs

```diff
@@ -231,5 +241,358 @@ public IEnumerator TestClothExport()
 
             TestAbcImported(exportPath);
         }
+        //Create And Delete
+        [UnityTest]
+        public IEnumerator TestCreateAndDelete()
+        {
+            // open scene
+            SceneManagement.EditorSceneManager.LoadScene(cd_scene, UnityEngine.SceneManagement.LoadSceneMode.Single);
+
+            // yield once while scene loads
+            yield return null;
+
+            yield return new WaitForSeconds(9f);
```

Comment:
why are you waiting for 9 seconds?

Reply:
I think it's better/safer if you turn the "auto export on play" option off in the exporter in the scene and explicitly call BeginRecording(). Also make sure to set the export path to a random abc file that will get deleted at the end of the test